src/locales/en.json

```
@@ -0,0 +1,30 @@
+{
+  "label": {
```

Biome (2.4.6)

[error] The key label was already declared.

(lint/suspicious/noDuplicateObjectKeys)